media/js/firefox/accounts.js

```diff
@@ -20,6 +24,17 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // check for fxaEmail in sessionStorage or URL params
+    try {
+        fxaEmail = sessionStorage.getItem('fxa-email');
+        sessionStorage.removeItem('fxa-email');
+    } catch(ex) {}
+
+    // if email address provided, send to FxA frame
+    if (fxaEmail) {
```

Comment:
Should we also add the same basic email validation here on this side, just for good measure?